src/codegeneration/generator/GeneratorTransformer.js

```diff
@@ -96,8 +96,6 @@ export class GeneratorTransformer extends CPSTransformer {
       ({expression, machine} = this.expressionToStateMachine(tree.expression));
     } else {
       expression = this.transformAny(tree.expression);
-      if (!expression)
-        expression = createUndefinedExpression();
```

Comment:
I gather that then new version always returns an expression here, so we don't have to handle `!expression` ? (I don't see changes where it is used below either).